.github/workflows/ci.yml

```diff
@@ -5,40 +5,42 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-lint-modules-${{ hashFiles('**/yarn.lock') }}
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
-      - run: yarn install
+          node-version: 20
+      - run: yarn install --frozen-lockfile --ignore-scripts
       - run: yarn run lint
 
   test:
+    needs: lint
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os:
+          - ubuntu-latest
         node-version:
-          - 14.x
-          - 16.x
+          - 14
+          - 16
```

Comment:
Might as well add 18 and 20 while you're at it.

Reply:
The tests seem to fail on 18 and 20, which is why I left them out (failing to retrieve a resource)

Reply:
Ok, but maybe don't bump the linting task to 20 either then.